bosh-lite.html.md.erb

```diff
@@ -49,7 +49,7 @@ Follow below steps to get it running on locally on VirtualBox:
       -o ~/workspace/bosh-deployment/bosh-lite-runc.yml \
       -o ~/workspace/bosh-deployment/jumpbox-user.yml \
       --vars-store ./creds.yml \
-      -v director_name="Bosh Lite Director" \
```

Comment:
I suppose this means secrets will be regenerated when the user next deploys.

If we're doing dashes, should we do lower case, too. And if we're changing the name... I wonder if we should remove the "director" suffix since that's kind of implied?

Reply:
>I suppose this means secrets will be regenerated when the user next deploys.
I guess you are referring to an upgrade/update of the bosh-lite environment and it's repercussions.

> if we're doing dashes, should we do lower case, too. And if we're changing the name... I wonder if we should remove the "director" suffix since that's kind of implied?
I am happy with any name compatible with director variable naming rules :)
BTW, all the other IAAS bootstrap guidelines use the name bosh-1. Eg. http://bosh.io/docs/init-aws.html#deploy

Reply:
this was kept this way for cf-release compatibility.

Reply:
What cf-release compatibility? Is it okay to change it?

Reply:
https://github.com/cloudfoundry/cf-release/blob/v277/scripts/generate-bosh-lite-dev-manifest#L27 comes to mind.

Reply:
@cppforlife, okay to merge and break compatibility? I see `EXPECTED_DIRECTOR_NAME` is a thing, so there's at least a workaround.

Reply:
I don't think there's any place to override that `EXPECTED_DIRECTOR_NAME` variable, though: it's not an expression like `EXPECTED_DIRECTOR_NAME=${EXPECTED_DIRECTOR_NAME:-Bosh Lite Director}`. /cc @dsabeti

Reply:
Oh, you're right; I read that too quickly. Thanks

Reply:
If this important to do, we could probably make that script smart enough to handle both names. It wouldn't have to be any smarter than something this (sorry for lack of bash fu):
```bash
EXPECTED_DIRECTOR_NAME_OLD="Bosh Lite Director"
EXPECTED_DIRECTOR_NAME_NEW="bosh-1"

ACTUAL_DIRECTOR_NAME="$(echo "$BOSH_STATUS" | grep Name)"

if [[ "$ACTUAL_DIRECTOR_NAME" != *"$EXPECTED_DIRECTOR_NAME_OLD"* ]] || [["$ACTUAL_DIRECTOR_NAME" != *"$EXPECTED_DIRECTOR_NAME_NEW"* ]]; then
  echo "Can only target $EXPECTED_DIRECTOR_NAME_OLD or $EXPECTED_DIRECTOR_NAME_NEW. Please use 'bosh target' before running this script."
  exit 1
fi
```